### DISCUSSION TITLE
q-drawer individually scrollable

### DISCUSSION THREAD

**Discussion by lisadejonghe** [quasarframework/quasar#12878] [General - Components / Directives / etc] [answered]
I have a header, drawer and page content. The header is in another file then the drawer and content. 

```
  <Toolbar/>
  <router-view/>
```

When scrolling the page, the header sticks to the top, but the drawer and page content both scroll while the drawer must stick.

```   
      <div >
         <SideBar show-if-above class="full-height" />
         <div>
           <Content />
         </div>
      </div>
```

When changing the inside `div` to `q-scroll-area`, the Content disappears. How can I make the sidebar be sticky and make the content scrollable?

**Comment by MichaelJCole** [comment]
Did you find an answer to this?

**Comment by suleimanas** [comment, marked as answer]
[understanding-the-view-prop](https://quasar.dev/layout/layout#understanding-the-view-prop)

**Comment by MichaelJCole** [comment]
Specifically, change the upper/lower case in the layout string: "The letters shown above are also case sensitive. For example, using at least one “L” (uppercase character instead of lowercase) will make your layout left side (drawer) be in a fixed position. Same applies for “H” (header), “F” (footer) and finally “R” (right side / drawer)."
